GoogleDataTransport/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased (v8.1.0)
+- Expose upload URLs which FirebasePerformance will depend upon.
+
 # v8.0.2
```

Comment:
8.0.2 is also unreleased, so both of these items should now be under 8.1.0

Reply:
Fixed.